jobs/gtfs-schedule-validator/README.md

```diff
@@ -22,3 +22,14 @@ available version of the validator. Instead, we use extract dates to determine w
 version of the validator was correct to use at the time the data was created. That way,
 we don't "punish" older data for not conforming to expectations that changed in the time
 since data creation.
+
+## Upgrading the Schedule Validator Version tips
+If you run into trouble when adding the new validator jar, it's because the default set for check-added-large-files in our pre-commit config which is a relatively low 500Kb. It's more meant as an alarm for local development than as an enforcement mechanism.
+You can make one commit that adds the jar and temporarily adds a higher file size threshold to the pre-commit config [like this one](https://github.com/cal-itp/data-infra/pull/2893/commits/7d40c81f2f5a2622123d4ac5dbbb064eb35565c6) and then a second commit that removes the threshold modification [like this one](https://github.com/cal-itp/data-infra/pull/2893/commits/1ec4e4a1f30ac95b9c0edffcf1f2b12e53e40733). That'll get the file through.
+
+Remember you need to rebuild and push the latest docker file to dhcr before changes will be reflected in airflow runs.
+
+You will need to parse the rules.json from the mobility validator.  [Here is a gist to help](https://gist.github.com/vevetron/7d4bbebd2f1d524728d5349293906e3a).
```

Comment:
To avoid depending on individually-owned resources or contractor-owned resources, we tend to prefer to move examples like this into the Cal-ITP GitHub organization. It could be its own standalone thing within the GitHub org (I believe gists aren't supported at the org level, so it'd be its own repo), it could be an example file within the `/jobs/gtfs-schedule-validator` subfolder, or it could be a code block within the README itself, but it likely should be somewhere besides a personal gist.

Reply:
Makes sense!  I'll modify in another PR.